docs/content/docs/connectors/datastream/elasticsearch.md

```diff
@@ -226,17 +226,60 @@ To use fault tolerant Elasticsearch Sinks, checkpointing of the topology needs t
 
 {{< tabs "d00d1e93-4844-40d7-b0ec-9ec37e73145e" >}}
 {{< tab "Java" >}}
+
+Elasticsearch 6:
+```java
+final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+env.enableCheckpointing(5000); // checkpoint every 5000 msecs
+
+Elasticsearch6SinkBuilder sinkBuilder = new Elasticsearch6SinkBuilder<String>()
+    .setDeliveryGuarantee(DeliveryGuarantee.AT_LEAST_ONCE)
+    .setHosts(new HttpHost("127.0.0.1", 9200, "http"))
+    .setEmitter(
+    (element, context, indexer) -> 
+    indexer.add(createIndexRequest(element)));
```

Comment:
This doesn't match with the JavaDoc. I'm not sure that the Chinese documentation is correct, based on FLINK-25589 and FLINK-26634.

Reply:
Whether this commit changes an error doc file? Should we change the English doc but change the Chinese doc? https://github.com/apache/flink-connector-elasticsearch/commit/3c9430593d9d61254b63a66234d8dba9c269060b

Reply:
I found the Chinese translations of `IMPORTANT: Checkpointing is not enabled by default but  ....... ` was deleted and the example of the fault tolerance section in the English doc wasn't updated in this former commit.

Reply:
The documentation at https://nightlies.apache.org/flink/flink-docs-master/api/java/org/apache/flink/connector/elasticsearch/sink/Elasticsearch6SinkBuilder.html is correct from what I've understood